docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md

```diff
@@ -0,0 +1,99 @@
+# 审计日志
+
+Nebula Graph 的审计日志功能可以将 Graph 服务接受到的所有操作进行分类存储，用户可以根据需要，追踪指定类型的操作。
+
+!!! enterpriseonly
+
+    仅企业版支持本功能。
+
+## 日志类别
+
+|类别|说明|
+|:--|:--|
+|`LOGIN` |客户端尝试连接 Graph 服务时，记录相关信息。 |
+|`EXIT`  | 断开与 Graph 服务的连接时，记录相关信息。 |
+|`DDL` |记录 DDL 语句的信息，包括元数据的变化信息。例如`CREATE SPACE`、`ALTER TAG`、`REBUILD INDEX`等。 |
+|`DQL` |记录 DQL 语句的信息，包括查询、排序等信息。例如`MATCH`、`GO`、`GET SUBGRAPH`、`ORDER BY`、`YIELD`、`RETURN`等。|
+|`DML` |记录 DML 语句的信息，包括实际数据的变化信息。例如`INSERT VERTEX`、`INSERT EDGE`、`DELETE EDGE`等。 |
+|`DCL`|记录 DCL 语句的信息，包括用户创建授权、快照、作业等运维信息。例如`CREATE USER`、`CREATE SNAPSHOT`、`ADD LISTENER`、`BALANCE`等。|
+|`UTIL`|记录工具类语句的信息，包括`SHOW`、`DESCRIBE`、`USE`、`SIGN IN`、`KILL QUERY`、`INGEST`等语句。 |
```

Comment:
Does each category not need to specify which statements are recorded to the user?